.github/workflows/ci.yml

```
@@ -173,4 +217,4 @@ jobs:
 #          PAYLOAD_SECRET: ${{ secrets.PAYLOAD_SECRET }}
 #        run: |
 #          echo "🚀 Deploying image blog-payload:${{ github.sha }}..."
-#          ./scripts/deploy.sh blog-payload:${{ github.sha }}
+#          ./scripts/deploy.sh blog-payload:${{ github.sha }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)